ardupilot_gz_bringup/launch/robots/wildthumper.launch.py

```diff
@@ -0,0 +1,199 @@
+                        "launch",
+                        "sitl_dds_udp.launch.py",
+                    ]
+                ),
+            ]
+        ),
+        launch_arguments={
+            "transport": "udp4",
+            "refs": PathJoinSubstitution(
+                [
+                    FindPackageShare("ardupilot_sitl"),
+                    "config",
+                    "dds_xrce_profile.xml",
+                ]
+            ),
+            "port": "2019",
+            "command": "ardurover",
+            "synthetic_clock": "True",
+            "wipe": "False",
+            "model": "json",
+            "speedup": "1",
+            "slave": "0",
+            "instance": "0",
+            "defaults": os.path.join(
+                pkg_ardupilot_sitl,
+                "config",
+                "default_params",
+                "rover-skid.parm",
+            )
+            + ","
+            + os.path.join(
+                pkg_ardupilot_sitl,
+                "config",
+                "default_params",
+                "dds_udp.parm",
+            ),
+            "sim_address": "127.0.0.1",
+            "master": "tcp:127.0.0.1:5760",
+            "sitl": "127.0.0.1:5501",
+        }.items(),
+    )
+
+    # Robot description.
+
+    # Ensure `SDF_PATH` is populated as `sdformat_urdf`` uses this rather
+    # than `GZ_SIM_RESOURCE_PATH` to locate resources.
+    if "GZ_SIM_RESOURCE_PATH" in os.environ:
+        gz_sim_resource_path = os.environ["GZ_SIM_RESOURCE_PATH"]
+
+        if "SDF_PATH" in os.environ:
+            sdf_path = os.environ["SDF_PATH"]
+            os.environ["SDF_PATH"] = sdf_path + ":" + gz_sim_resource_path
+        else:
+            os.environ["SDF_PATH"] = gz_sim_resource_path
+
+    # Load SDF file.
+    sdf_file = os.path.join(
+        pkg_ardupilot_sitl_models, "models", "wildthumper", "model.sdf"
+    )
+    with open(sdf_file, "r") as infp:
+        robot_desc = infp.read()
+
+        # substitute `models://` with `package://ardupilot_sitl_models/models/`
+        # for sdformat_urdf plugin used by robot_state_publisher
+        robot_desc = robot_desc.replace(
+            "model://wildthumper",
+            "package://ardupilot_sitl_models/models/wildthumper")
+        # print(robot_desc)
+
+    # Publish /tf and /tf_static.
+    robot_state_publisher = Node(
+        package="robot_state_publisher",
+        executable="robot_state_publisher",
+        name="robot_state_publisher",
+        output="both",
+        parameters=[
+            {"robot_description": robot_desc},
+            {"frame_prefix": ""},
+        ],
+    )
+
+    # Bridge.
+    bridge = Node(
+        package="ros_gz_bridge",
+        executable="parameter_bridge",
+        parameters=[
+            {
+                "config_file": os.path.join(
+                    pkg_project_bringup, "config", "wildthumper_bridge.yaml"
+                ),
+                "qos_overrides./tf_static.publisher.durability": "transient_local",
```

Comment:
What is the reason this needs to be modified? I thought all tf_static publishers should be transient local by default.

Reply:
LM check. Most likely this is carried over from another example.

Reply:
Move to issue: https://github.com/ArduPilot/ardupilot_gz/issues/36. The settings are used in multiple examples and should be updated consistently.